koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -637,11 +637,11 @@ def check_whitelist(self, name, target_info):
     def runBuilds(self, src, target_info, arches, output_template,
                   scratch=False, yum_repourls=None, branch=None, push_url=None):
         subtasks = {}
-        if koji.util.multi_fnmatch(arch, self.options.literal_task_arches):
-            taskarch = arch
-        else:
-            taskarch = koji.canonArch(arch)
         for arch in arches:
+            if koji.util.multi_fnmatch(arch, self.options.literal_task_arches):
+                taskarch = arch
+            else:
+                taskarch = koji.canonArch(arch)
```

Comment:
It's not, this actually should have been merged already afaik, I don't know why it's showing up in my changeset. I cherry-pick'd it because it was causing me issues due to how the koji tags are setup in Fedora and koji-containerbuild kept trying to build armv7hl containers